src/Pages/SignUp/SignUp.js

```diff
@@ -4,11 +4,11 @@ import { useAuthContext } from '../../Context'
 
 function SignUp() {
 
-  const { signup } = useAuthContext()
+  const { signUp } = useAuthContext()
   const [email, setemail] = useState('');
   const [password, setpassword] = useState('')
-  const [confirmpassword, setconfirmpassword] = useState('')
-  const [errorvalues, seterrorvalues] = useState({})
+  const [confirmPassword, setconfirmPassword] = useState('')
```

Comment:
if you are following camelCase. then expected naming would be 
const [confirmPassword, setConfirmPassword]

Reply:
thanks, i used lowecase camel convention, fixed it